### ISSUE TITLE
 Review Design Wireframe: Contact Us Page

### ISSUE THREAD

**Issue by lem31** [NoroffFEU/musikkforandrerliv.no#90]
- [x] Visual Hierarchy
- [x] Layout
- [x] Consistency
- [x] Screen size
- [x] Usability Principles
- [x] Web Content Accessibility Guidelines (WCAG)
- [x] Colors
- [x] Fonts
- [x] Whitespace and padding
- [x] Image quality
- [x] Iconography
- [x] Micro interactions
- [x] Error states and feedback
- [x] Performance Optimization
- [x] Design documentation

See Design wireframe/prototype checklist for more info

**Comment by EliNygard** [comment]
- Section Our Mission desktop view: The title could align with the top line of the image, buttons could align with the bottom line of the image. Or the space should be equal between the top margin of the title and on the bottom margin of the buttons. Both options to archive better symmetry.  
-

**Comment by EliNygard** [comment]
Review finished and comments added for the Design Team

**Comment by Vickabu** [comment]
_Contact form 'Send' button on desktop and mobile view: Should follow the style guide for buttons and have uppercase letters_-fixed
_Mobile view: No need to have the empty white space between the 'Our mission' section and 'Contact form'. Does not look so good._-redused spacing, but not removed case of the sectioning and spacing style.

**Comment by EliNygard** [comment]
Okay, great. Thank you! Closing this.
